WordPress/Classes/ViewRelated/Pages/BasePageListCell.h

```diff
@@ -1,19 +1,33 @@
 #import <UIKit/UIKit.h>
 #import "WPPostContentViewProvider.h"
 
+@class AbstractPost;
+@class BasePageListCell;
 
-@protocol BasePageListCellDelegate <NSObject>
-@optional
-- (void)cell:(nonnull UITableViewCell *)cell receivedMenuActionFromButton:(nonnull UIButton *)button forProvider:(nonnull id<WPPostContentViewProvider>)contentProvider;
-- (void)cell:(nonnull UITableViewCell *)cell receivedRestoreActionForProvider:(nonnull id<WPPostContentViewProvider>)contentProvider;
-@end
-
+/// A block that represents an action triggered by tapping on a button in a cell.
+///
+/// @param      cell        The cell that contains the button that was tapped.
+/// @param      button      The button that was tapped.
+/// @param      post        The post represented by the cell that was tapped.
+///
+typedef void(^BasePageListCellActionBlock)(BasePageListCell* _Nonnull cell,
```

Comment:
Oh yeah, of course! The only way around it I can think of would be to mark the header in an assume nonnull block: https://gist.github.com/frosty/5a2e478680c84a660f6cb40e31b83ab0 – happy for you to leave it as it is if you like though!